ground/gcs/src/plugins/coreplugin/generalsettings.cpp

```diff
@@ -161,12 +167,46 @@ void GeneralSettings::apply()
     m_useSessionManaging = m_page->cbSessionMessaging->isChecked();
     m_autoConnect = m_page->checkAutoConnect->isChecked();
     m_autoSelect = m_page->checkAutoSelect->isChecked();
-    m_proxyType = m_page->proxyTypeCB->itemData(m_page->proxyTypeCB->currentIndex()).toInt();
+    m_proxyType = m_page->proxyTypeCB->currentData().toInt();
     m_proxyPort = m_page->portLE->text().toInt();
     m_proxyHostname = m_page->hostNameLE->text();
     m_proxyUser = m_page->userLE->text();
     m_proxyPassword = m_page->passwordLE->text();
     QNetworkProxy::setApplicationProxy (getNetworkProxy());
+    if(!m_usePortableSettings && m_page->cb_usePortableSettings->isChecked()) {
```

Comment:
Just to make sure I understand-- to use portable settings, you need to start up once with global settings and then either import or change this?